docs/cody/clients/feature-reference.mdx

```diff
@@ -134,7 +134,6 @@
 | @-file               | ✅           | ✅             | ❌          | ❌       |
 | @-symbol             | ✅           | ❌             | ❌          | ❌       |
 | Admin LLM selection  | ✅           | ✅             | ✅          | ✅       |
-| Users LLM selection  | ✅           | ✅             | ❌          | ❌       |
```

Comment:
maybe we can keep this but put coming soon instead?

Reply:
I think let's save it for the time it's there. :)